### DISCUSSION TITLE
Can't get a custom snippet to load for Markdown

### DISCUSSION THREAD

**Discussion by SathyaBhat** [LazyVim/LazyVim#4360] [Q&A] [answered]
I'm asking this after reading multiple discussions on how to get custom snippets to work and I can't get mine to work. Here's my nvim version:

```
❯ nvim -v
NVIM v0.10.1
Build type: Release
LuaJIT 2.1.1723675123
```

lazyvim release: 12.38.2.

Here's my `~/.config/nvim` structure 

```bash
├── init.lua
├── lazy-lock.json
├── lazyvim.json
├── LICENSE
├── lua
│   ├── config
│   │   ├── autocmds.lua
│   │   ├── keymaps.lua
│   │   ├── lazy.lua
│   │   └── options.lua
│   └── plugins
│       └── example.lua
├── package.json
├── README.md
├── snippets
│   └── hugo.json
└── stylua.toml
```

Essentially, it's the standard Lazyvim post install structure with no changes, except for the `package.json` which looks like:

```json

cat package.json
{
  "contributes": {
    "snippets": [
      {
        "language": "markdown",
        "path": "./snippets/hugo.json"
      }
    ]
  }
}
```

and `./snippets/hugo.json` looks like 

```json
cat ./snippets/hugo.json
{
  "hugo fancybox": {
    "prefix": "hfb",
    "body": ["fancybox12345"],
    "description": "Fancybox"
  }
}
```

However, when I load a `.md`  file and type `hfb`, the snippet doesn't come up - how can I debug this further and find out why it doesn't pop up? The other snippets from `friendly-snippets` work fine 

![image](https://github.com/user-attachments/assets/ffaeb78a-2159-4a71-8526-ef90fd55c92e)

**Comment by dpetka2001** [comment, marked as answer]
It has to be called `markdown.json` and you don't need the `package.json` any more (it's only used for `friendly-snippets`, so just delete it).

**Reply by SathyaBhat** [reply]
Wow, that did it. thank you so much, appreciate the help. Is there someplace where I can read up on these changes?

Thank you

**Reply by dpetka2001** [reply]
I had a similar problem in the past and I just asked at `nvim-snippets` repo. This is [the issue](https://github.com/garymjr/nvim-snippets/issues/33). Other than that just read through the PRs and issues at the repo.

**Reply by SathyaBhat** [reply]
understood, thanks again.
